src/main/java/org/fjala/gugumber/salesforce/entities/Event.java

```diff
@@ -12,93 +12,263 @@
  */
 public class Event {
-    private User assignedToUser;
+
+    /**
+     * Saves value of variable "Assigned To", to create an Event.
+     */
+    private String assignedToUser;
+
+    /**
+     * Saves value of variable "Subject", to create an Event.
+     */
     private String subject;
-    private Contact name;
-    private Account relatedToAccount;
+
+    /**
+     * Saves value of variable "Name", to create an Event.
+     */
+    private String nameContact;
+
+    /**
+     * Saves value of variable "Related To", to create an Event.
+     */
+    private String relatedToAccount;
+
+    /**
+     * Saves value of variable "Location", to create an Event.
+     */
     private String location;
-    private String startDate;
-    private String endDate;
-    private String Description;
 
-    public User getAssignedToUser() {
+    /**
+     * Saves value of variable "Start Date", to create an Event.
+     */
+    private Date startDate;
+
+    /**
+     * Saves value of variable "Start Time", to create an Event.
+     */
+    private String startTime;
+
+    /**
+     * Saves value of variable "End Date", to create an Event.
+     */
+    private Date endDate;
+
+    /**
+     * Saves value of variable "End Time", to create an Event.
+     */
+    private String endTime;
+
+    /**
+     * Saves value of variable "Description", to create an Event.
+     */
+    private String description;
+
+    /**
+     * Gets the value assigned to "assignedTo" variable.
+     *
+     * @return a string, this is "assignedTo" variable.
+     */
+    public String getAssignedToUser() {
         return assignedToUser;
     }
 
-    public void setAssignedToUser(User assignedToUser) {
+    /**
+     * Sets the value "assignedTo" variable.
+     */
+    public void setAssignedToUser(final String assignedToUser) {
         this.assignedToUser = assignedToUser;
     }
 
+    /**
+     * Gets the value assigned to "subject" variable.
+     *
+     * @return a string, this is "Subject" variable.
+     */
     public String getSubject() {
         return subject;
     }
 
-    public void setSubject(String subject) {
+    /**
+     * Sets the value "subject" variable.
+     */
+    public void setSubject(final String subject) {
         this.subject = subject;
     }
 
-    public Contact getName() {
-        return name;
+    /**
+     * Gets the value assigned to "nameContact" variable.
+     *
+     * @return a string, this is "nameContact" variable.
+     */
+    public String getNameContact() {
+        return nameContact;
     }
 
-    public void setName(Contact name) {
-        this.name = name;
+    /**
+     * Sets the value "nameContact" variable.
+     */
+    public void setNameContact(final String nameContact) {
+        this.nameContact = nameContact;
     }
 
-    public Account getRelatedToAccount() {
+    /**
+     * Gets the value assigned to "relatedToAccount" variable.
+     *
+     * @return a string, this is "relatedToAccount" variable.
```

Comment:
should be by example:
```suggestion
     * @return a the value of relatedToAccount as string.
```